scripts/install_linux.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/sh
+
+# Script to install the Docker ACI integration CLI on Ubuntu (Beta).
+
+set -eu
+
+RELEASE_URL=https://api.github.com/repos/docker/aci-integration-beta/releases/latest
+LINK_NAME="${LINK_NAME:-com.docker.cli}"
+DRY_RUN="${DRY_RUN:-}"
+
+desktop_install_url="https://www.docker.com/products/docker-desktop"
+engine_install_url="https://docs.docker.com/get-docker/"
+
+link_path="/usr/local/bin/${LINK_NAME}"
+existing_cli_path="/usr/bin/docker"
+
+manual_install() {
+	echo "Please follow the manual install instructions"
+}
+
+is_new_cli() {
+	azure_version_str="$($1 version 2>/dev/null | grep 'Azure' || true)"
+	if [ -n "$azure_version_str" ]; then
+		echo 1
+	else
+		echo 0
+	fi
+}
+
+echo "Running checks..."
+
+# Check OS
+if [ "$(command -v uname)" ]; then
+	case "$(uname -s)" in
+		"Linux")
+			# Check for Ubuntu/Debian based distro
+			if ! [ -f "/etc/lsb-release" ]; then
+				echo "Warning: This script has been tested on Ubuntu and may not work on other distributions"
+			fi
+			# Pass
+			;;
+		"Darwin")
+			echo "Error: Script not needed on macOS, please install Docker Desktop Edge: $desktop_install_url"
+			exit 1
+			;;
+		"*")
+			echo "Error: Unsupported OS, please follow manual instructions"
+			exit 1
+			;;
+	esac
+else
+	# Assume Windows
+	echo "Error: Script not needed on Windows, please install Docker Desktop Edge: $desktop_install_url"
+	exit 1
+fi
+
+user="$(id -un 2>/dev/null || true)"
+sh_c='sh -c'
+sudo_sh_c='sh -c'
+if [ "$user" != 'root' ]; then
+    if [ "$(command -v sudo)" ]; then
+        sudo_sh_c='sudo -E sh -c'
+    elif [ "$(command -v su)" ]; then
+        sudo_sh_c='su -c'
+    else
+        echo "Error: This installer needs the ability to run commands as root."
+        exit 1
+    fi
+fi
+
+if [ -n "$DRY_RUN" ]; then
+	sh_c='echo $sh_c'
+	sudo_sh_c='echo $sudo_sh_c'
+fi
+
+# Check if Docker Engine is installed
+if ! [ "$(command -v docker)" ]; then
+	echo "Error: Docker Engine not found"
+	echo "You need to install Docker first: $engine_install_url"
+	exit 1
+fi
+
+download_cmd='curl -fsSLo'
+# Check that system has curl installed
+if ! [ "$(command -v curl)" ]; then
+	echo "Error: curl not found"
+	echo "Please install curl"
+	exit 1
+fi
+
+DOWNLOAD_URL=$(curl -s ${RELEASE_URL} | grep "browser_download_url.*docker-linux-amd64" | cut -d : -f 2,3)
+
+# Check if the ACI CLI is already installed
+if [ $(is_new_cli "docker") -eq 1 ]; then
+	echo "You already have the Docker ACI Integration CLI installed, overriding with latest version"
+	download_dir=$($sh_c 'mktemp -d')
+	$sh_c "${download_cmd} ${download_dir}/docker-aci ${DOWNLOAD_URL}"
```

Comment:
Would be good to factor this with the download code [lower down](https://github.com/docker/aci-integration-beta/pull/17/files#diff-65f46e4d078e4e0901b0680970a6d659R146-R152). Can be done in a follow up.